src/main/java/org/sonar/plugins/ldap/windows/AdConnectionHelper.java

```diff
@@ -79,6 +79,11 @@
    * Attribute for storing common name of a user in an active directory
    */
   public static final String COMMON_NAME_ATTRIBUTE = "cn";
+  
+  /**
+   * Attribute for storing common name of a user in an active directory
+   */
+  public static final String REAL_NAME_ATTRIBUTE = "displayName";
```

Comment:
Remove this field if we are going to get the User Real Name attribute from WindowsAuthSettings.